worlds/wl4/test/test_helpers.py

```diff
@@ -0,0 +1,44 @@
+from . import WL4TestBase
+
+from .. import items, locations
+from ..types import ItemType, Passage
+
+main_levels = ['Palm Tree Paradise', 'Wildflower Fields', 'Mystic Lake', 'Monsoon Jungle',
+               'The Curious Factory', 'The Toxic Landfill', '40 Below Fridge', 'Pinball Zone',
+               'Toy Block Tower', 'The Big Board', 'Doodle Woods', 'Domino Row',
+               'Crescent Moon Village', 'Arabian Night', 'Fiery Cavern', 'Hotel Horror']
+
+class TestHelpers(WL4TestBase):
+
+    def test_item_filter(self):
+        '''Ensure item filters and item names match.'''
+        with self.subTest('Jewel Pieces'):
+            pieces = items.filter_items(type=ItemType.JEWEL)
+            assert all(map(lambda p: p[0].endswith('Piece'), pieces))
+            assert all(map(lambda p: p[1].type, pieces))
```

Comment:
This test fails for me. This passes though:
```suggestion
            assert all(map(lambda p: p[1].type == ItemType.JEWEL, pieces))
```